tests/test.rs

```diff
@@ -9,7 +9,11 @@ fn compile_fail_tests() {
     let t = trybuild::TestCases::new();
     t.compile_fail("tests/compile-fail/*.rs");
 }
-
+#[test]
+fn pass_tests() {
+    let t = trybuild::TestCases::new();
+    t.pass("tests/pass/*.rs");
```

Comment:
I'm not a huge fan of this structure. Why not just have them all in `tests/`?

Reply:
removed this, tests moved to test.rs